src/Compiler/Directory.Packages.props

```diff
@@ -19,5 +19,11 @@
     <PackageVersion Include="System.Text.Json" Version="$(SystemTextJsonVersion)" />
     <PackageVersion Include="xunit.assert" Version="$(XunitAssertVersion)" />
     <PackageVersion Include="xunit.extensibility.execution" Version="$(XunitExtensibilityExecutionVersion)" />
+
+    <!-- Temporarily force analyzers to match compiler version -->
```

Comment:
Tracking issue for the temporary?

Reply:
https://github.com/dotnet/razor-tooling/issues/6758